File: launchdarkly/typescript-node/src/GetDestinationExample.ts
import * as fs from 'fs';
import api from "launchdarkly_client"
import models from "launchdarkly_client"

const apiCaller = new api.DataExportDestinationsApi();
apiCaller.setApiKey(api.DataExportDestinationsApiApiKeys.ApiKey, "YOUR_API_KEY");

apiCaller.getDestination(
  "projectKey_string", // projectKey
  "environmentKey_string", // environmentKey
  "id_string", // id
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling DataExportDestinationsApi#getDestination:");
  console.log(error.body);
});

File: launchdarkly/kotlin/src/main/kotlin/oseg/launchdarkly_examples/GetDestinationExample.kt
package oseg.launchdarkly_examples

import com.launchdarkly.client.infrastructure.*
import com.launchdarkly.client.apis.*
import com.launchdarkly.client.models.*

import java.io.File
import java.time.LocalDate
import java.time.OffsetDateTime
import kotlin.collections.ArrayList
import kotlin.collections.List
import kotlin.collections.Map
import com.squareup.moshi.adapter

@ExperimentalStdlibApi
class GetDestinationExample
{
    fun getDestination()
    {
        ApiClient.apiKey["Authorization"] = "YOUR_API_KEY"

        try
        {
            val response = DataExportDestinationsApi().getDestination(
                projectKey = null,
                environmentKey = null,
                id = null,
            )

            println(response)
        } catch (e: ClientException) {
            println("4xx response calling DataExportDestinationsApi#getDestination")
            e.printStackTrace()
        } catch (e: ServerException) {
            println("5xx response calling DataExportDestinationsApi#getDestination")
            e.printStackTrace()
        }
    }
}
